source/elements/oneDAL/source/algorithms/decomposition/pca.rst

```diff
@@ -32,7 +32,31 @@ oneDAL specifies two methods for PCA computation:
 -----------------------
 Covariance-based method
 -----------------------
-[TBD]
+
+This method uses eigendecomposition of the covariance matrix to define the
+principal components of the dataset. Following steps are distinguished:
+
+#. Covariance computation.
+#. Eigenvectors and eigenvalues computation.
+#. Resulting matrix formation. 
+
+Covariance matrix computation should be in the following way:
```

Comment:
"be performed in the following way" maybe? Or think about rephrasing this

Not should, but shall?

Reply:
fixed